t/t2201-job-cmd.t

```diff
@@ -317,6 +320,7 @@ test_expect_success 'flux job: the queue contains active jobs' '
 # be any running jobs to kill
 
 test_expect_success 'flux job: killall with no args works' '
+	flux jobs &&
```

Comment:
was this addition accidental?

Reply:
Yeah, that must have been debugging. I'll take care of it.

Reply:
Ok, squashed and gone. Thanks!!